Source/Icebreaker/Bot/IcebreakerBot.cs

```diff
@@ -314,6 +325,42 @@ private async Task HandleMessageActivityAsync(ITurnContext turnContext, Cancella
             }
         }
 
+         /// <summary>
+        /// Handle profile updates from users.
+        /// </summary>
+        /// <param name="activity">Message from submitted card</param>
+        /// <param name="turnContext">Context object containing information cached for a single turn of conversation with a user.</param>
+        /// <param name="cancellationToken">Propagates notification that operations should be canceled.</param>
+        /// <returns>Tracking Task</returns>
+        private async Task OnProfileSaveAsync(Activity activity, ITurnContext turnContext, CancellationToken cancellationToken)
+        {
+            var cardPayload = JToken.Parse(activity.Value.ToString());
+            var cardType = cardPayload["action"].Value<string>().ToLowerInvariant();
```

Comment:
Please rename to something denoting it is the 'action' such as `cardActionType` or `cardAction`